File: Treinamento/Python/Exercícios/Estrutura_de_repetição/Questão02.py
print('*** Cadastro ***')
nome = str(input('Digite o nome desejado: '))
senha = str(input('Digite a senha desejada: '))

while senha == nome:
    print('A senha não pode ser igual o nome, digite novamente')
    senha = str(input('Digite a senha desejada: '))

print('Nome do usuário: ' + str(nome))
print('Senha do usuário: ' + str(senha))

File: Treinamento/Java/Estrutura_de_repetição/Questão02.java
import java.util.Scanner;
public class Questão02 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o nome desejado: ");
        String nome = s.nextLine();

        System.out.print("Digite a senha desejada: ");
        String senha = s.next();

        while (senha.equalsIgnoreCase(nome.replaceAll(" ", ""))) {
            System.out.println("Senha inválida, digite novamente");
            System.out.print("Digite a senha desejada: ");
            senha = s.next();
        }

        System.out.println("Nome do usuário: " + nome + "\nSenha: " + senha);
    }
}
            
